### PR TITLE
William/rn67

### PR DESCRIPTION
**Pull request by swansontec**
This PR gets the app to build without issues on an x86 computer. I believe it will allow the app to build without issues on an M1, once we [fix react-native-mymonero-core](https://github.com/EdgeApp/react-native-mymonero-core/pull/6) and get rid of react-native-fast-crypto.

#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [x] Tested on iOS Tablet
- [x] Tested on small Android
- [ ] n/a

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201773165509953

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -102,6 +104,3 @@ buck-out/
 
 # CocoaPods
 /ios/Pods/
-
```
Comment (swansontec) [inline .gitignore:105]: Put this back in the "Edge additions" section.

File: package.json [hunk 2]
```diff
@@ -63,6 +63,7 @@
     "codecov": "3.5.0",
     "core-js": "2.5.3",
     "create-hash": "1.2.0",
+    "deprecated-react-native-listview": "0.0.7",
```
Comment (swansontec) [inline package.json:66]: Redundant line. Run sort-package-json.

**Review by swansontec — COMMENTED** [review]
Noticed some small things.

**Review by thehobbit85 — APPROVED** [review]

**Review by thehobbit85 — APPROVED** [review]
Approved. Resolve conflicts and merge.

**Pull request merged**